app/src/main/java/com/github/fribourgsdp/radio/User.kt

```diff
@@ -28,6 +35,11 @@ class User (val name: String, val color: Int) {
                 Random.nextInt(100, 200)
     }
 
+    fun save(context: Context){
+        val userFile = File(context.filesDir, USER_DATA_PATH)
+        userFile.writeText(Json.encodeToString(this))
+    }
+
```

Comment:
How persistent is this storage? Will this saved user state be kept when the app is closed?
All good for me!

Reply:
yes, it is called app-specific storage: it is stored locally in the device but only accessible to your own app